### PR TITLE
add and access server guilds from config

### PR DESCRIPTION
**Pull request by Sooraj-beep**
Guild ids are now accessed through config files. The config template has been updated to reflect this. Overall readme update will be done in another PR.

### CODE REVIEW

**Comment by Sooraj-beep** [comment]
test

File: bot.py [hunk 1]
```diff
@@ -22,8 +24,8 @@
 )
 
 
-GUILDS = [discord.Object(id=1137183815949881357), discord.Object(id=402891511991369740)]
-
+__cfg = get_config().get("general", None)
```
Comment (aarctan) [inline bot.py:27]: perhaps default to the ualberta cs server instead of none

Comment (aarctan) [reply bot.py:27]: nvm just realized this is something else

File: bot.py [hunk 2]
```diff
@@ -22,8 +24,8 @@
...
+GUILDS = [discord.Object(id=gid) for gid in __cfg.get("guild_ids", [])] if __cfg else []
```
Comment (aarctan) [inline bot.py]: default to cs server

**Review by aarctan — APPROVED** [review]

**Pull request merged**